src/js/components/DataTable/DataTable.js

```diff
@@ -406,7 +406,11 @@ const DataTable = ({
           )}
         </StyledDataTable>
       </OverflowContainer>
-      {paginate && items && <Pagination alignSelf="end" {...paginationProps} />}
+      {paginate && items && items.length ? (
+        <Pagination alignSelf="end" {...paginationProps} />
+      ) : (
+        undefined
+      )}
```

Comment:
This would also be valid and result the same:

```suggestion
      {paginate && items && !!items.length && (
        <Pagination alignSelf="end" {...paginationProps} /> 
      )}
```

Or by convention, I think Grommet has tended to use `null` instead of `undefined` in these circumstances.

```suggestion
      {paginate && items && items.length ? (
        <Pagination alignSelf="end" {...paginationProps} />
      ) : null}
```

Reply:
Thanks! I'll update to `null` to align with common Grommet convention.